modelcenter/PP-LCNetV2/introduction_en.ipynb

```diff
@@ -0,0 +1,239 @@
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/201664376-e3cdd4ac-0767-4773-a6cc-b6885796bc1e.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "#### 4.2.3 Shortcut\n",
+    "\n",
+    "The residual structure has been widely used by many models since it was proposed, but in lightweight convolutional neural networks, the elite-wise addition operation brought by the residual structure will affect the speed of the model. In PP-LCNetV2, The effect of the residual structure on the model was tested with the Stage as the unit, and it was found that the use of the residual structure did not necessarily improve the performance, so PPLCNetV2 only used the residual structure in the last Stage: add Shortcut in the Block, as shown in the figure below.\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/201664472-42139a13-28a1-43b9-92d7-d355f10ed56d.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "#### 4.2.4 Activation Function\n",
+    "\n",
+    "In the current lightweight convolutional neural networks, ReLU and Hard-Swish activation functions are most commonly used. Although Hard-Swish is usually better in terms of model performance, we found that some inference platforms have unsatisfactory efficiency optimization for Hard-Swish activation functions. Therefore, in order to give consideration to universality, PP-LCNetV2 uses the ReLU activation function by default, and our tests show that the ReLU activation function has little impact on the performance of larger models.\n",
+    "\n",
+    "\n",
+    "#### 4.2.5 SE\n",
+    "\n",
+    "Although SE module can significantly improve model performance, its impact on model speed should not be ignored. In PP-LCNetV1, we found that the use of SE module in the back of the model can obtain the maximum benefit. In the optimization process of PP-LCNetV2, we did further experiments on the position of SE module by Stage, and found that better balance could be achieved by using it in Stage4."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Notes\n",
+    "The delay test of PP-LCNet series model at the Intel CPU was completed based on the Intel Xeon Gold 6148. PaddlePaddle was used in the inference framework, MKLDNN is turned on, and the batch size was 1 and the number of threads was 10."
```

Comment:
delay -> inference

Reply:
PP-LCNet -> PP-LCNetV2

Reply:
Intel Xeon Gold 6271C

Reply:
-> The inference speed of PP-LCNet series model were tested at Intel Xeon Gold 6271C CPU with PaddlePaddle inference framework, the batch size was 1 and the number of threads was 10.